addons/account/tests/test_account_account.py

```diff
@@ -480,6 +480,10 @@ def create_account_group(name, code_prefix, company):
         # Delete group_101 and recreate it
         group_101.unlink()
         group_101 = create_account_group('group_101', 101, self.env.company)
+        group_3 = create_account_group('group_3', 3, self.env.company)
+        group_31 = create_account_group('group_31', 31, self.env.company)
+        with self.assertRaises(ValidationError), self.cr.savepoint():
+            group_3.code_prefix_start = 312
```

Comment:
I would expect this to make `group_31` a root, with one leaf as `group_3`, not cause an error

Reply:
Hello @william-andre , should i adapt this functionality ?  Your input will valuable